frontend/src/store/episode/reducer.ts

```diff
@@ -10,7 +10,6 @@ import {
   updateCommentsAfterDelete,
```

Comment:
After all this changes we have unexpected result. After double click on unliked comment we receive 2 likes at once.